csrc/quantization/fp8/per_token_group_quant_fp8.cu

```diff
@@ -0,0 +1,126 @@
+#include "common.cuh"
+#include "dispatch_utils.h"
+
+#include <ATen/cuda/CUDAContext.h>
+#include <c10/cuda/CUDAGuard.h>
+
+#ifndef USE_ROCM
+  #include <cub/cub.cuh>
+#else
+  #include <hipcub/hipcub.hpp>
+#endif
+
+template <typename scalar_t, bool is_column_major>
+__global__ void per_token_group_quant_fp8_kernel(
+    const scalar_t* __restrict__ input, FP8_TYPE* __restrict__ output_q,
+    float* __restrict__ output_s, const int group_size,
+    const int groups_per_row, const int num_rows, const int y_s_stride) {
+  float const min_scaling_factor = 1.0f / (FP8_E4M3_MAX * 512.f);
```

Comment:
where does the 512.f come from?

Reply:
I copied this from `dynamic_per_token_scaled_fp8_quant_kernel` https://github.com/vllm-project/vllm/blob/f89978ad7cf3732a4a482db8e784aba7e417abbd/csrc/quantization/fp8/common.cu#L33